src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -200,7 +200,8 @@ public UserTenant update(UUID consortiumId, UserTenant primary) {
   }
 
   private User prepareShadowUser(UUID userId, UserTenantEntity userTenantEntity, FolioExecutionContext folioExecutionContext) {
-    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(userTenantEntity.getTenant().getId(), folioExecutionContext))) {
+    try (var context = new FolioExecutionContextSetter(prepareContextForTenant(userTenantEntity.getTenant()
+      .getId(), folioExecutionContext))) {
```

Comment:
not very obvious formatting

Reply:
I am testing force split if line longer than max width function. looks like is not working. I will fix